### PR TITLE
LRAC-13975 [F] The current interests of an individual have an underline

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://issues.liferay.com/browse/LRAC-13975

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/profile/components/InterestsCard.jsx [hunk 1]
```diff
@@ -31,7 +30,8 @@ export const InterestsList = ({channelId, groupId, id, interests}) => (
 			<ListGroup.Item className='interest' key={name}>
 				<ListGroup.ItemTitle className='text-truncate'>
 					{name ? (
-						<Link
+						<ClayLink
+							decoration='none'
 							to={toRoute(
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/profile/components/InterestsCard.jsx:35]: Hey @fmarcio, ClayLink doesn't have `to` attr. Please change it to use `href` instead.

**Review by interaminense — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/profile/components/InterestsCard.jsx (hunk 1 above)
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/individual/profile/components/InterestsCard.jsx]: @interaminense done!

**Comment by fmarcio** [comment]
ci:test:sf

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @fmarcio

**Comment by fmarcio** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by interaminense** [comment]
ci:stop

**Comment by ilzamcmed** [comment]
ci:stop

**Comment by ilzamcmed** [comment]
ci:stop:force

**Comment by interaminense** [comment]
Resent https://github.com/liferay-ac/liferay-portal/pull/506

**Pull request closed without merging**